integration-tests/src/test/java/eu/tsystems/mms/tic/testframework/test/guielement/GuiElementAdditionalTests.java

```diff
@@ -23,45 +23,46 @@
 package eu.tsystems.mms.tic.testframework.test.guielement;
 
 import eu.tsystems.mms.tic.testframework.AbstractTestSitesTest;
-import eu.tsystems.mms.tic.testframework.core.utils.LogAssertUtils;
+import eu.tsystems.mms.tic.testframework.core.utils.Log4jFileReader;
 import eu.tsystems.mms.tic.testframework.exceptions.ElementNotFoundException;
 import eu.tsystems.mms.tic.testframework.exceptions.TimeoutException;
 import eu.tsystems.mms.tic.testframework.pageobjects.GuiElement;
 import eu.tsystems.mms.tic.testframework.pageobjects.Locate;
 import eu.tsystems.mms.tic.testframework.utils.FileUtils;
 import eu.tsystems.mms.tic.testframework.webdrivermanager.WebDriverManager;
-import java.io.File;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import java.io.File;
+import java.util.List;
+
 public class GuiElementAdditionalTests extends AbstractTestSitesTest {
 
+    private final Log4jFileReader LOG_4_J_FILE_READER = new Log4jFileReader("logs/pre-test-log4j.log");
+
     @Test
     public void testT01_Upload() {
         final WebDriver driver = WebDriverManager.getWebDriver();
         final File resourceFile = FileUtils.getResourceFile("testfiles/Test.txt");
         final String absoluteFilePath = resourceFile.getAbsolutePath();
 
-
         GuiElement input = new GuiElement(driver, By.id("2"));
         input.sendKeys(absoluteFilePath);
     }
 
+    // Needs pretest `io.testerra.test.pretest_guielement.testGuiElement_Create_SensibleData()`
```

Comment:
`/** Needs pretest {@link GuiElementAdditional2Tests#testGuiElement_Create_SensibleData}. */` is _much more_ concise, checked by IDE linter and generates a clickable link when rendered by IDEs mouse-over.